common/src/main/java/org/opensearch/ml/common/transport/model/MLInPlaceUpdateModelAction.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.common.transport.model;
+
+import org.opensearch.action.ActionType;
+
+public class MLInPlaceUpdateModelAction extends ActionType<MLInPlaceUpdateModelNodesResponse> {
+    public static final MLInPlaceUpdateModelAction INSTANCE = new MLInPlaceUpdateModelAction();
```

Comment:
Do we need a separate update model action? Should this inplace update be part of regular model update? May be we can offer this inplace update for remote models?

Reply:
We have to, because in-place update required us to sent request to every specific node whereas index update does not require that. These are two different behavior and cannot use same request/response class to describe them.

Reply:
You mean in this class, we are updating the cache not the index?

Reply:
I think we've all agreed in-place update is a making sense name in the design review meeting. However we can always refactor this into something like `InCacheUpdate` if you'd prefer, but this may take a lot of additional efforts.

Reply:
Meanwhile this is an internal API, so we can always refactor it if we want

Reply:
Refactored